lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -627,6 +627,21 @@ def __add__(self, other):
         new_group.extend(other)
         return new_group
 
+    def _get_track_by_id(self, python_id):
```

Comment:
would it make sense to turn this into a `__contains__` method (and the parameter is the track obj instead of the id) so you can use the `in` operator?

Reply:
I think it would make a lot of sense to also add a `__contains__` method for convenience (and I will add it).

This method in particular has something to do with communication with another software where we can't store the entire object for other implementational reasons. This is also why I chose to make it private.

Reply:
Actually,  come to think of it, I don't think we actually need a `__contains__` dunder. The default suffices for this container already (since we defined `__iter__` and just iterating over the elements and against them should suffice).

Reply:
makes sense! 👍